problems/euler/problem1/java/LoganPappas/Driver.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+
+ public class Driver {
+
+     /**
+      * @param args the command line arguments
```

Comment:
+1  Code is well written with comments. I'm nitpicking but the only change that you could make is to remove the "param" flag from the top, it would just make it look a little cleaner as is not needed.

Reply:
You could also update your README to include instructions on how to run the code from the command line instead of an IDE.